docs/en/04_Changelogs/beta/4.6.0-beta1.md

```diff
@@ -73,7 +73,7 @@ module, you should upgrade it as well.
 
 `silverstripe/mimevalidator` is now a core module and will ship by default on new projects. Projects referencing `silverstripe/recipe-core` will automatically install `silverstripe/mimevalidator` when they upgrade to 4.6.0.
 
-Read [Allowed file types](Developer_Guides/Files/Allowed_file_types) in the Silverstripe CMS doc for all the details.
+Read [Allowed file types](https://docs.silverstripe.org/en/4/developer_guides/files/allowed_file_types) in the Silverstripe CMS doc for all the details.
```

Comment:
`/Developer_Guides/Files/Allowed_file_types` should work. I would rather double check that otherwise if we ever change domain or path, we'll be stuck fixing every link every where.

Reply:
Just confirmed that the problem was the missing `/` in front of the url.

If you look at the "See Recipes for more information" link on this [4.5.2 changelog](https://docs.silverstripe.org/en/4/changelogs/4.5.2/), it works just fine and it's written as `[Recipes](/getting_started/recipes)` in the MD file.

I'll do a PR to remove the absolute link.